components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -182,6 +182,25 @@ class RewardsServiceImpl : public RewardsService,
 
   void GetAddressesForPaymentId(const GetAddressesCallback& callback) override;
 
+  void SetCatalogIssuers(const std::string& json) override;
```

Comment:
these should all be listed under `// LedgerClient impl`

Reply:
Just checked, and it is now listed under so I gather you made this change